source/GameInterface/Services/Towns/Commands/TownDebugCommand.cs

```diff
@@ -103,6 +103,7 @@ public static string Info(List<string> args)
             return $"ID: '{args[0]}' not found";
         }
 
+        Fief fief = town.Settlement.SettlementComponent as Fief;
```

Comment:
Identation is wrong here?

Reply:
I've made a change, don't know if it was not identate well, but it had a space above, maybe it is that ?